pkcs11test.cc

```diff
@@ -83,6 +83,7 @@ void usage() {
   cerr << "  -m name : name of PKCS#11 library" << endl;
   cerr << "  -l path : path to PKCS#11 library" << endl;
   cerr << "  -s id   : slot ID to perform tests against" << endl;
+  cerr << "  -S n    : slot index (0 for first slot)" << endl;
```

Comment:
Could we use either "index" or "offset" everywhere?  (I'd probably lean towards "index")

Reply:
I think index too. I will change to that